src/api/java/baritone/api/utils/Helper.java

```diff
@@ -34,13 +38,17 @@
      */
     Helper HELPER = new Helper() {};
 
-    ITextComponent MESSAGE_PREFIX = new TextComponentString(String.format(
-            "%s[%sBaritone%s]%s",
-            TextFormatting.DARK_PURPLE,
-            TextFormatting.LIGHT_PURPLE,
-            TextFormatting.DARK_PURPLE,
-            TextFormatting.GRAY
-    ));
+    static ITextComponent getPrefix() {
+        ITextComponent baritone = new TextComponentString(BaritoneAPI.getSettings().shortBaritonePrefix.value ? "B" : "Baritone");
```

Comment:
because that's gross, also logging would have those codes. this way it logs as just `[Baritone]` not with the §

Reply:
I am very sure it will still just have `§` added anyways and if it dont then im crying because the Minecraft color code plugin for the game output is gonna be broken now.

Reply:
> the Minecraft color code plugin for the game output is gonna be broken now

dude, components are used everywhere... not just for the baritone prefix